cli/src/types.rs

```diff
@@ -111,6 +111,15 @@ pub struct RejectionSource {
     pub reason: Option<String>,
 }
 
+/// Locksmith token details accessible after creation.
+#[derive(Serialize, Deserialize, PartialEq, Eq, Clone, Debug)]
+pub struct UserToken {
+    pub name: String,
+    pub creation_time: DateTime<Utc>,
+    pub access_time: Option<DateTime<Utc>>,
+    pub expiry: Option<DateTime<Utc>>,
```

Comment:
Is this just optional because we haven't deployed expiry to prod? Are we going to come back later and remove the `Option` here?

Reply:
Yeah basically just to be safe. I don't think there's any drawbacks to having it optional though.